vip-consumer/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.12-slim
+
+WORKDIR /code
+
+COPY ./requirements.txt /code/requirements.txt
+
+RUN pip install --no-cache-dir --upgrade -r /code/requirements.txt
+
+COPY . .
+COPY .env .env
```

Comment:
you can't copy the env file,  its best practise to inject the env file when building

Reply:
You inject the env file to the code and then use it inside the docker.